### ISSUE TITLE
Add option to disable the shortcut

### ISSUE THREAD

**Issue by Kern3l** [Infocatcher/Private_Tab#193]
In Polish we use [Alt] with many keys to get our special letters and many times I accidentally enter private mode while typing. It's so troublesome that it may make me uninstall this awesome plugin eventually.

**Comment by Infocatcher** [comment]
You can disable or modify shortcuts, see https://github.com/Infocatcher/Private_Tab#keyboard-shortcuts

**Comment by Kern3l** [comment]
Thanks. It should be an option in normal (now not existing) plugin configuration.

**Comment by Infocatcher** [comment]
Yes, this way is better, but it's too difficult to write entire configuration interface. :(
And unfortunately there is no API about keyboard shortcuts (moreover: now used manual code to handle keyboard events due to non-restartless native things).
